sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/GpuShuffleExchangeExecBase.scala

```diff
@@ -172,6 +172,9 @@ abstract class GpuShuffleExchangeExecBase(
   import GpuMetric._
 
   private lazy val useKudo = RapidsConf.SHUFFLE_KUDO_SERIALIZER_ENABLED.get(child.conf)
+  private lazy val kudoMeasureBufferCopy = RapidsConf
```

Comment:
Could we please consider another name for this variable?  I quite like the convention used for the conf key:
```suggestion
  private lazy val kudoBufferCopyMeasurementEnabled = RapidsConf
```